### PR TITLE
Go mono

### PR DESCRIPTION
**Pull request by ilkerde**
This change-set enables coypu to be built on Mono / Linux.

I created this because I want to build and execute coypu-driven tests on Linux. While tapping into the repository, I had a hard time to get albacore running on Linux (strange, eh?). So I ditched albacore in favor of _fake_ and _paket_. I ported over the `rakefile.rb` to make use of `fakefile.fsx`.

Apart of that, of course, I fixed minor issues in tests (all which were functional).

### CODE REVIEW

File: src/Coypu.AcceptanceTests/WaitAndRetryExamples.cs [hunk 1]
```diff
@@ -38,8 +38,8 @@ public void SetUp()
         protected void ApplyAsyncDelay()
         {
             // Hide the HTML then bring back after a short delay to test robustness
-            browser.ExecuteScript("window.holdIt = window.document.body.innerHTML;");
             browser.ExecuteScript("window.document.body.innerHTML = '';");
+            browser.ExecuteScript("window.holdIt = window.document.body.innerHTML;");
```
Comment (adiel) [inline src/Coypu.AcceptanceTests/WaitAndRetryExamples.cs]: This seems weird, could you explain?

Comment (ilkerde) [inline src/Coypu.AcceptanceTests/WaitAndRetryExamples.cs]: I am verry sorry, I must have mixed those lines while trying to get the `FindAllCss` failing tests to pass. I executed the tests in both versions, they fail on both.

Consequently I reverted this change now. It indeed looks weird.

While investigating, I spotted that `fakefile.fsx` had a `TestAll` target defined which chained all other test argets together. I removed it in favor of independently executing test categories.

**Comment by adiel** [comment]
Sorry for not responding for a while. I've got a lot of failing tests on windows for this, not had time to investigate why yet, so can't merge it yet.

**Comment by forki** [comment]
I recommend to add AppVeyor and Travis CI builds along with this pull request.

**Comment by obstar** [comment]
this project will go slowly toward next .net core version, the next version of coypu will be compatible with .net standard, also this PR is 3yr old.

**Pull request closed without merging**